shared/src/business/entities/Scan.ts

```diff
@@ -72,19 +72,17 @@ export class Scan extends JoiValidationEntity {
     return aggregatedPngs;
   }
 
-  static VALIDATION_ERROR_MESSAGES = {
-    batches: '#At least one batch is required',
-  };
-
-  getErrorToMessageMap() {
-    return Scan.VALIDATION_ERROR_MESSAGES;
-  }
+  static VALIDATION_RULES = joi.object().keys({
+    batches: joi
+      .array()
+      .min(1)
+      .required()
+      .messages({ '*': '#At least one batch is required' }),
```

Comment:
![thinking](https://media.giphy.com/media/XbPFv2ihC3rRm/giphy.gif)

Is it just me or does the '#' look like a typo? I see it was already in the error message before but that just looks funky to me.

Reply:
I was thinking the same. Wanted to ask you if I should just remove it